com.trollworks.gcs/src/com/trollworks/gcs/preferences/ReferenceLookupPreferences.java

```diff
@@ -43,14 +55,25 @@
      */
     public ReferenceLookupPreferences(PreferencesWindow owner) {
         super(I18n.Text("Page References"), owner);
-        setLayout(new BorderLayout());
+        setLayout(new PrecisionLayout().setColumns(2));
+        Preferences prefs = Preferences.getInstance();
+
+        //Command line to use to open PDFs
```

Comment:
Again, I don't find these comments useful. If you think they provide value, I'm fine with you leaving them here, but in that case, add a space after the //

This is likely something the formatter will do for you automatically, but I don't recall off-hand.